spring-boot-admin/files/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM openjdk:8-jdk-alpine
```

Comment:
Do we need this file?

Reply:
Not sure I understand this comment? you mean the line (for multi stage docker file or what)? results in a smaller image.

Reply:
There's a Dockerfile in the parent dir that is one you crafted.  What's the purpose of this one?